fp.js

```diff
@@ -1,28 +1,74 @@
 // Convert the following code to use `.map`
 
-var peoplesAges = []
+// var peoplesAges = []
 var people = [
   {name: "Jane", age: 32},
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person){
-  peoplesAges.push(person.age)
-});
+// people.forEach(function(person){
+//   peoplesAges.push(person.age)
+// });
+//THis is the template from my notes on the subject.
+// var numbers = [1, 4, 9, 3, 27, 7];
+// var roots = numbers.map(Math.sqrt);
+//this is what I found on w3schools for an almost identical example, except they do not use the .map:
+// var persons = [
+//     {firstname : "Malcom", lastname: "Reynolds"},
+//     {firstname : "Kaylee", lastname: "Frye"},
+//     {firstname : "Jayne", lastname: "Cobb"}
+// ];
+//
+//
+// function getFullName(item, index) {
+//     var fullname = [item.firstname,item.lastname].join(" ");
+//     return fullname;
+// }
+//
+// function myFunction() {
+//     document.getElementById("demo").innerHTML = persons.map(getFullName);
+// }
+var peoplesAges = people.map(people.age);
```

Comment:
`.map` needs to receive a function as an argument -- not just the value you are trying to access. So the solution looks something like this...

```js
var peoplesAges = people.map(function(person) => {  // where `person` represents each item in `people`
  return person.age  // the new array will contain whatever the function returns
})
```